credentials/utopia-natcert-vcdm-v2/credential.json

```diff
@@ -2,7 +2,6 @@
   "@context": [
     "https://www.w3.org/ns/credentials/v2",
     "https://w3id.org/citizenship/v4rc1",
-    "https://w3id.org/security/data-integrity/v2",
```

Comment:
Good catch. It'd be nice to have a tool that shows which contexts have no terms from them used--or better yet, which terms came from which context file.